sc/social/like/browser/images/ico-email.svg

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I updated the issue, yesterday when we fixed the colors @agnogueira suggested to change the icon.

Please keep this file separated from CSS now, we will mix it when use webpack.